libs-scala/test-evidence/tag/src/scala/com/daml/test/evidence/tag/FuncTest.scala

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.test.evidence.tag
+
+/** A functional test case.
+  *
+  * Captures a test for a functional requirement in addition to the non-functional requirements such as security or
+  * reliability.
+  *
+  * TODO(soren): Refine data captured in a functional test case tag.
```

Comment:
MINOR: remove/replace TODOs

Reply:
@skisel-da please rename these todo's to `TODO test evidencing:`
We might need to discuss with Soren whether he believes some of them can be removed, but let's not block this PR on that.